pkg/network/network.go

```diff
@@ -25,14 +25,15 @@ import (
 	"golang.org/x/sys/unix"
 
 	"github.com/jackpal/gateway"
-	"github.com/nubificus/urunc/internal/log"
+	"github.com/sirupsen/logrus"
 	"github.com/vishvananda/netlink"
 )
 
 const DefaultInterface = "eth0"
```

Comment:
Add a FIXME comment about supporting more interfaces, maybe point it to the relevant issue (if we don't have, we should create one). The same stands for the name of the veth container endpoint (eth0) -- maybe we need to discover that (again, as the kata people do, by examining the flags of all interfaces in the netns (https://github.com/kata-containers/kata-containers/blob/3b2fb6a604a624f1ec0c21a85d6e4182344aac4c/src/runtime/virtcontainers/network_linux.go#L337)

Reply:
Done